Lib/asyncio/streams.py

```diff
@@ -108,6 +116,11 @@ def connect(host=None, port=None, *,
 
 
 def connect_read_pipe(pipe, *, limit=_DEFAULT_LIMIT):
+    """Takes a file-like object *pipe* to return a Stream object of the mode
```

Comment:
Please make sure that all docstrings follow this: First paragraph is 80 characters long or less.  It should provide a quick summary of what the method/function is for.  Later paragraphs can elaborate on that further.

Reply:
Done to here and other docstrings I have added to make sure first sentence have a short description.